include/play.js

```diff
@@ -1,227 +1,187 @@
+          queue.muted = !queue.muted;
+          if (queue.muted) {
+            queue.resource.volume.setVolumeLogarithmic(0);
+            queue.textChannel.send(i18n.__mf("play.mutedSong", { author: user })).catch(console.error);
+          } else {
+            queue.resource.volume.setVolumeLogarithmic(queue.volume / 100);
+            queue.textChannel.send(i18n.__mf("play.unmutedSong", { author: user })).catch(console.error);
+          }
+          break;
+
+        case "🔉":
+          reaction.users.remove(user).catch(console.error);
+          if (queue.volume == 0) return;
+          if (!canModifyQueue(member, queue)) return i18n.__("common.errorNotChannel");
+          queue.volume = Math.max(queue.volume - 10, 0);
+          queue.resource.volume.setVolumeLogarithmic(queue.volume / 100);
+          queue.textChannel
+            .send(i18n.__mf("play.decreasedVolume", { author: user, volume: queue.volume }))
+            .catch(console.error);
+          break;
+
+        case "🔊":
+          reaction.users.remove(user).catch(console.error);
+          if (queue.volume == 100) return;
+          if (!canModifyQueue(member, queue)) return i18n.__("common.errorNotChannel");
+          queue.volume = Math.min(queue.volume + 10, 100);
+          queue.resource.volume.setVolumeLogarithmic(queue.volume / 100);
+          queue.textChannel
+            .send(i18n.__mf("play.increasedVolume", { author: user, volume: queue.volume }))
+            .catch(console.error);
+          break;
+
+        case "🔁":
+          reaction.users.remove(user).catch(console.error);
+          await message.client.commands.get("loop").execute(message);
+          break;
+
+        case "🔀":
+          reaction.users.remove(user).catch(console.error);
+          await message.client.commands.get("shuffle").execute(message);
+          break;
+
+        case "⏹":
+          reaction.users.remove(user).catch(console.error);
+          await message.client.commands.get("stop").execute(message);
+          collector.stop();
+          break;
+
+        default:
+          reaction.users.remove(user).catch(console.error);
+          break;
       }
     });
 
-    queue.connection.subscribe(queue.player);
+    collector.on("end", () => {
+      playingMessage.reactions.removeAll().catch(console.error);
 
-    if (!silent) {
-      try {
-        var playingMessage = await queue.textChannel.send(
-          i18n.__mf("play.startedPlaying", { title: song.title, url: song.url })
-        );
-        await playingMessage.react("⏭");
-        await playingMessage.react("⏯");
-        await playingMessage.react("🔇");
-        await playingMessage.react("🔉");
-        await playingMessage.react("🔊");
-        await playingMessage.react("🔁");
-        await playingMessage.react("🔀");
-        await playingMessage.react("⏹");
-      } catch (error) {
-        console.error(error);
+      if (PRUNING && !playingMessage?.deleted) {
```

Comment:
`!playingMessage?.deleted` should be changed into `playingMessage?.deletable`, to silence deprecation messages from discord.js in the console.